### PR TITLE
Plan change support not working for auto renewal setup

### LINKED ISSUE #430: Plan change support not working for auto-renewal setup
Plan change support not working for auto-renewal setup

- ENTITLEMENT_PLAN_CHANGE_REQUESTED: If policy has auto-approve on, than the procurement plan change can be auto-approved. If auto-approve is off, do nothing.
- ENTITLEMENT_PLAN_CHANGED: Once this event is received, make the plan policy permission changes.

### PR DESCRIPTION
**Pull request by mservidio**
Fixes #430, #432, #399

### All Submissions:

* [X] Have you followed the guidelines in our Contributing document?
* [X] Have you checked to ensure there aren't other open [Pull Requests](../../../pulls) for the same update/change?

### New Feature Submissions:

1. [X] Does your submission pass tests?
2. [X] Have you lint your code locally prior to submission?

### CODE REVIEW

**Review by swilliams11 — APPROVED** [review]
LGTM

**Comment by mservidio** [comment]
@phriscage Still a work in progress (testing), but this should do it for the entitlement sync changes for now to take care of the plan changes.

**Review by swilliams11 — APPROVED** [review]
LGTM

File: frontend/src/components/EditAccount.vue [hunk 1]
```diff
@@ -127,6 +127,7 @@ extend('email', {
 });
 
 import Dialog from '@/components/Dialog.vue';
+import config from '../config';
```
Comment (phriscage) [inline frontend/src/components/EditAccount.vue:130]: Should this be `_config` and have a `computed` function below to return `config` (like the other view component updates)?

Comment (mservidio) [reply frontend/src/components/EditAccount.vue:130]: In this particular case it wasn't necessary, as it's only used through the script code. I ran into conflicts with the naming when I used the name 'config' and used the variable through UX template code.

File: frontend/src/config.js [hunk 2]
```diff
@@ -81,6 +81,23 @@ class Config {
...
+      return m;
+    }
+  }
+
+  get userGuideUrl() {
+    return this.getConfigValue('VUE_APP_USER_GUIDE_URL', false);
+  }
+
+  get githubUrl() {
+    return this.getConfigValue('VUE_APP_GITHUB_URL', false);
```
Comment (phriscage) [inline frontend/src/config.js:99]: Would the default value for these be a blank string `""` instead of boolean false?

Comment (mservidio) [reply frontend/src/config.js:99]: The second param indicates if the getConfigValue function should throw or not if the key doesn't exist:

`getConfigValue(key, throws = true) {
    let val;
    if (process.env.NODE_ENV === 'development') {
      val = process.env[key] || this.config[key];
    } else {
      val = this.config[key];
    }
    if (throws && !val) {
      throw new Error(`Configuration missing for key: ${key}`);
    }
    return val;
  }`

**Pull request merged**